solid_attenuator/ioc_kfe_at1k4_calc/__main__.py

```diff
@@ -0,0 +1,64 @@
+import os
+import sys
+
+from caproto.server import ioc_arg_parser, run
+
+from .. import util
+from ..sxr import create_ioc
+
+FIRST_FILTER = 1
+NUM_BLADES = 4
+
+
+if '--production' in sys.argv:
```

Comment:
This is the only implementation that seems kind of weird, but it's totally fine

Reply:
Yeah, I wasn't so happy about it. Wonder what a better way to split default dev/prod configuration would be.

Reply:
I think command-line argument is fine, it just seems messy to directly check `sys.argv` on import instead of doing something like `argparse` on execution. But since it's so simple it's probably better/fine to do it this way.